receiver/vcenterreceiver/factory.go

```diff
@@ -28,11 +27,9 @@ func NewFactory() receiver.Factory {
 
 func createDefaultConfig() component.Config {
 	return &Config{
-		ScraperControllerSettings: scraperhelper.ScraperControllerSettings{
-			CollectionInterval: 2 * time.Minute,
```

Comment:
At a minimum, I do not think we should shorten this one. Experience tells me that this is not a good default for this API.